gzipstream/tests/test_gzipstreamfile.py

```diff
@@ -0,0 +1,23 @@
+from .. import gzipstreamfile
+import unittest
+
+GzipStreamFile = gzipstreamfile.GzipStreamFile
+
+
+class TestSequenceFunctions(unittest.TestCase):
```

Comment:
Just noticed these tests are not run in Travis. What is the best way to make the test runner pick them up?